flow/project.py

```diff
@@ -3487,7 +3488,7 @@ def _main_exec(self, args):
             jobs = self
         try:
             try:
-                operation_function = self._operation_functions[args.operation]
+                operation_function = self._operations[args.operation]
             except KeyError:
                 operation = self._operations[args.operation]
```

Comment:
There is no need for the `try` `except` block any more.

Reply:
@b-butler I removed the block.